controller.go

```diff
@@ -35,14 +38,22 @@ import (
 const (
 	defaultInvitationEmailTemplate = `Hello developer of great software, Kubernetes engineer or fellow human,
 
-A user of APPUiO Cloud has invited you to join them. Follow https://portal.dev/invitations/{{.Invitation.ObjectMeta.Name}}?token={{.Invitation.Status.Token}} to accept this invitation.
+A user of APPUiO Cloud has invited you to join them. Follow https://portal.dev/invitations/{{.Object.ObjectMeta.Name}}?token={{.Object.Status.Token}} to accept this invitation.
```

Comment:
While I think it's fine to adjust the email template rendering to use `Object` as the input, we'll need to keep this in mind for the rollout, since we override the invitation email template in the cluster configuration at the moment.